src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -568,6 +570,20 @@ private void validateRules() {
       isValid = false;
       Logger.log(Level.SEVERE, "randomSeed can't be less than %d!", MIN_RANDOM_SEED);
     }
+
+    // TODO: Fix so they don't check each other
```

Comment:
Right, because currently you'll get duplicate errors. Not immediately obvious to me how to solve this in an elegant way. Oh, how about each of these fields only checks other fields that follow it in a lexical sorting?

Reply:
Filed #352 to take care of this in the future, since it's a relatively minor issue.